airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -135,7 +148,30 @@ private boolean runProcess(final String jobId,
           Collections.emptyMap(),
           args);
 
-      LineGobbler.gobble(process.getInputStream(), LOGGER::info, CONTAINER_LOG_MDC_BUILDER);
+      try (final InputStream stdout = process.getInputStream()) {
+        // finds and collects any AirbyteMessages from stdout
+        // also builds a list of raw dbt errors and stores in streamFactory
+        airbyteMessagesByType = streamFactory.create(IOs.newBufferedReader(stdout))
+            .collect(Collectors.groupingBy(AirbyteMessage::getType));
+
+        // picks up error logs from dbt
+        String dbtErrorStack = String.join("\n\t", streamFactory.getDbtErrors());
+
+        if (!dbtErrorStack.equals("")) {
+          AirbyteMessage dbtTraceMessage = new AirbyteMessage()
+              .withType(Type.TRACE)
+              .withTrace(new AirbyteTraceMessage()
+                  .withType(AirbyteTraceMessage.Type.ERROR)
+                  .withEmittedAt((double) System.currentTimeMillis())
+                  .withError(new AirbyteErrorTraceMessage()
+                      .withFailureType(FailureType.SYSTEM_ERROR) // TODO: decide on best FailureType for this
```

Comment:
Maybe we should use `DATA_ERROR` or even more specifically `DBT_ERROR` here?

The reason for an error from dbt is not necessarily clear, e.g. it could be a problem with the source data **or** it could be a system error from a bug we've introduced **or** it could be an issue with the destination (and other **or**s)...